sources/constraint-declaration-validation.asciidoc

```diff
@@ -11,6 +11,9 @@ The Bean Validation specification defines a framework for declaring constraints
 
 Bean Validation also offers a way to declare constructor and method constraints where parameters and return values are the constrained elements. We will discuss method constraints declaration in detail in <<constraintdeclarationvalidationprocess-methodlevelconstraints>>.
 
+Furthermore, constraints can be applied to the elements of generic container types such as `Map`, `List` or `Optional`.
```

Comment:
I'm tempted to say that's it's closer to a "normal" declaration and this should be expressed before method level constraints.